tasks/cephfs/cephfs_test_case.py

```diff
@@ -15,11 +22,55 @@ class CephFSTestCase(unittest.TestCase):
     Handles resetting the cluster under test between tests.
     """
     # Environment references
+    mounts = None
+    fs = None
+    ctx = None
+
+    # FIXME weird explicit naming
```

Comment:
Actually it looks like maybe you did below with the "Create friendly mount_a, mount_b attrs" section?